src/Filter.js

```diff
@@ -16,42 +13,14 @@ function Filter({
   setPage,
 }) {
   const [copyDisabled, setCopyDisabled] = useState(false)
-  const newFilter = {}
-
-  if (filter && filter.where) Object.assign(newFilter, { where: filter.where })
-
-  const getTotalRows = new Query(`${dataset}_aggregate`)
-    .filter(newFilter)
-    .find(new Query('aggregate').find('count'))
-
-  // getTotalRows.filter({where: {ConnectedArea: {_eq: 'DK1'}}});
-  const QUERY = gql`
```

Comment:
@cotts this doesn't make sense to me. Why it is moved to `TableContainer` component?

Reply:
to keep both requests in the same component and also avoid filter input keep resetting after submit, if that still doesn't make sense, I can use the change to use in a new component to get the total and set there.

Reply:
@cotts we are using Apollo cache as a state and point is that to make requests from different components. Each component should make request it needs.

Reply:
@cotts ok, now I can see that `TableContainer` doesn't query for total rows anymore 👍  and that query is now made from `TotalRows` component which is exactly how it should be 👍